Cabal/src/Distribution/Simple/Program/GHC.hs

```diff
@@ -532,6 +532,7 @@ data GhcOptions = GhcOptions
   , ghcOptDynObjSuffix :: Flag String
   -- ^ only in 'GhcStaticAndDynamic' mode
   , ghcOptHiDir :: Flag FilePath
+  , ghcOptHieDir :: Flag (Maybe FilePath)
```

Comment:
`Flag` is already a kind of `Maybe`. Does it have to hold another `Maybe`?

Reply:
Indeed, thanks for pointing this out.